openedx/core/djangoapps/theming/templatetags/optional_include.py

```diff
@@ -5,14 +5,15 @@
 possible, in order to match the behavior of the {% include %} template tag,
 except for making it optional.
 """
-# Because we want to match the original loader_tags.py file as closely as
-# possible, we should disable pylint so it doesn't complain about the violations
+
 # that are already in that file
 # pylint: skip-file
+from __future__ import absolute_import
+
+# Because we want to match the original loader_tags.py file as closely as
+# possible, we should disable pylint so it doesn't complain about the violations
```

Comment:
It looks like `isort` mangled the multi-line comment here; this should stay before the comment lines above.

Reply:
OK, I'll fix it. Thank you!